### ISSUE TITLE
Intermittent Option::unwrap panic in mozbrowsericonchange_event.html.

### ISSUE THREAD

**Issue by Ms2ger** [servo/servo#8616]
```
  ▶ CRASH [expected OK] /_mozilla/mozilla/mozbrowser/mozbrowsericonchange_event.html
  │ 
  │ thread 'ScriptTask PipelineId { namespace_id: PipelineNamespaceId(0), index: PipelineIndex(0) }' panicked at 'called `Option::unwrap()` on a `None` value', src/libcore/option.rs:365
  │ stack backtrace:
  │    1:     0x5608e70c7510 - sys::backtrace::tracing::imp::write::h23bfa7a49ed99d87Klt
  │    2:     0x5608e70ca255 - panicking::log_panic::_<closure>::closure.39180
  │    3:     0x5608e70c9ca1 - panicking::log_panic::h846e3da4ec5bd0b1Wlx
  │    4:     0x5608e70b3953 - sys_common::unwind::begin_unwind_inner::h04ad996baf6f81c7bds
  │    5:     0x5608e70b4008 - sys_common::unwind::begin_unwind_fmt::h44d552a66c65968bhcs
  │    6:     0x5608e70c6be1 - rust_begin_unwind
  │    7:     0x5608e70f7c0f - panicking::panic_fmt::h54ca80e608a83d7dwZK
  │    8:     0x5608e70f38d8 - panicking::panic::h2a24d1d860d3cd423XK
  │    9:     0x5608e6199d97 - dom::window::_<impl>::handle_fire_timer::h7ffde7567cbe0189G7U
  │   10:     0x5608e6209410 - script_task::_<impl>::handle_msgs::_<closure>::closure.153274
  │   11:     0x5608e61edf53 - script_task::_<impl>::handle_msgs::hda52b03099a1eba7JrZ
  │   12:     0x5608e61b6cd7 - boxed::_<impl>::call_box::call_box::h9587974893088428059
  │   13:     0x5608e5d98e14 - sys_common::unwind::try::try_fn::try_fn::h9955666643315721214
  │   14:     0x5608e70c6a48 - __rust_try
  │   15:     0x5608e70c301b - sys_common::unwind::try::inner_try::h25c781202f1ad3a1J9r
  │   16:     0x5608e5d98fa4 - boxed::_<impl>::call_box::call_box::h16268556686401107631
  │   17:     0x5608e70c8d43 - sys::thread::_<impl>::new::thread_start::h89508aa1f401717d8Dw
  │   18:     0x7f087c4ad181 - start_thread
  │   19:     0x7f087ccc747c - __clone
  │   20:                0x0 - <unknown>
  └ Pipeline failed in hard-fail mode.  Crashing!
```

**Comment by jdm** [comment]
I assume this is
```
253         // Since the event id was the expected one, at least one timer should be due.
254         assert!(base_time >= self.timers.borrow().last().unwrap().next_call);
```
from ActiveTimers::fire_timer.

**Comment by vegayours** [comment]
I have fixed this in PR #8768, so we can close it.

**Comment by jdm** [comment]
:dancers:
